### ISSUE TITLE
Version 14.20.0.25 

### ISSUE THREAD

**Issue by greg84** [xamarin/xamarin-macios#12577]
### Steps to Reproduce

1. Attempt to use "Pair to Mac" functionality on Visual Studio for Windows.
2. Receive a message prompting you to download version 14.20.0.25 of Xamarin.iOS.

### Expected Behavior

The new version 14.20.0.25 downloads and installs automatically.
Pair to Mac works correctly.

### Actual Behavior

The version 14.20.0.25 failed to download, but I visited the URL in the error message and it downloaded successfully.

I now have 14.20.0.25 installed, but Visual Studio for Mac now keeps bugging me for to download the *older* version 14.20.0.24. The new version doesn't appear in the releases list either.

**Comment by spouliot** [comment]
WRT version numbers: the windows extension required a small change (that was not needed for macOS) and this is why it needs `14.20.0.25` instead of `14.20.0.24`. That happened too late in the process to sync both versions.

The pairing issue is Windows specific, added label + vsts issue https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1388891 for the XVS team

**Comment by m-sadegh-sh** [comment]
When this fix will be released? I'm blocked by this issue :( Any workaround?

**Comment by greg84** [comment]
@m-sadegh-sh You can download the Xamarin.iOS package manually from here:

https://aka.ms/xvs/pkg/macios/14.20.0.25

Then just ignore the warnings from VS that keep prompting you to downgrade.

**Comment by m-sadegh-sh** [comment]
@grep84 Thanks. Got no warning and everything is functioning properly.
